src/App.tsx

```diff
@@ -1,61 +1,129 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+import users from './api/users';
```

Comment:
The import `import users from './api/users';` is redundant because `usersFromServer` is already imported and used. Consider removing this line to clean up the code.

Reply:
> The import `import users from './api/users';` is redundant because `usersFromServer` is already imported and used. Consider removing this line to clean up the code.

Good point